addon/utils/has-block.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+const [major, minor] = Ember.VERSION.split('.');
+const isGlimmer = major > 2 || (major == 2 && minor >= 10); // >= 2.10
```

Comment:
Maybe this could be made into a util shared by `utils/get-mutable-attributes.js` and `utils/has-block`? Something like:

```javascript
import emberVersionInfo from './utils/ember-version-info';

const { major, minor, isGlimmer } = emberVersionInfo();
```

Reply:
Also, if we went this route, might be good to coerce `major` and `minor` into numbers.

Reply:
Yes I tried to do it but it's not easy to test. But it is still a good idea to have this :)